docker/keylime_agent.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=The Keylime compute agent
+Requires=tpm2-abrmd.service
+
+[Service]
```

Comment:
I think maybe we just let an OC based script start the agent/ and or registrar? Unless we want to have two different containers, one for the agent, one for the registrar?

Reply:
I was a little unclear about what would be running in a container. I thought it was just the agent. But if it's more than that we can definitely expand it.

Typically you want containers to not require anything outside of them to "do the right thing". If we have multiple similar containers we can solve that a couple of different ways:

1. Put all the common stuff in a base image and the role specific stuff in images on top of that.
2. Add a new entrypoint script which takes arguments based on what should be run under systemd. Then it could enable/disable the specific services we need for that role and then turn it over to systemd.

Reply:
Yeah you're completely right, I think ultimately it will just be the agent running so we may be able to leave it.  If anything we should go with option 2. just for debugging/ testing purposes.  Also, with the agent's being started by `systemd` how will we see the logs for that process? (might be a dumb question but will they be routed to the "logs" tab for openshift?)